src/lib/cid.test.js

```diff
@@ -0,0 +1,166 @@
+/* global describe it expect */
+// @ts-check
+import { toCidOrNull, getCodecOrNull, getCodeOrNull, toCidStrOrNull } from './cid'
+import * as multiformats from 'multiformats'
+import * as dagCbor from '@ipld/dag-cbor'
+import * as dagPb from '@ipld/dag-pb'
+import crypto from 'crypto'
+
+const { CID, hasher, bytes } = multiformats
+
+export const sha256 = hasher.from({
```

Comment:
sha256 is **_not_** exported from multiformats correctly even though it says I can import it from `multiformats/hashes/sha2`

<img width="733" alt="image" src="https://user-images.githubusercontent.com/1173416/178087333-ebafaa18-4d6f-4a45-b7c7-382097c6ff54.png">

Reply:
```js
import { sha256 } from 'multiformats/hashes/sha2'
```

if the docs are wrong, a PR would be appreciated

Reply:
That import doesn't even work :(

Reply:
what runtime or bundler is this using? perhaps something needs to be updated here to support proper export maps?

Reply:
react@16.14 is probably the culprit here, I don't think they'd nailed down their export maps functionality quite right back then but I don't know why this import specifically would fail tbh

but I bet upgrading react here would be a major pain too! so go with whatever works I suppose.

Reply:
I'm assuming this has something to do with migrating js-ipfs stuff to esm, but the GUI packages all still being cjs